get the 2D pipeline ready to run (#6)
WayScience/NF1_2D_organoid_profiling_pipeline: 1.max_projection/README.md

# Generate max-projection image sets

> In this module, we use python to perform max z projection and to get the middle most z-slice of the image sets.

Reviewer: ```suggestion
In this module, we use a Python implementation to perform max z projection and to get the middlemost z-slice of the image sets.
```
Recommend potentially including the package you used.
